src/quicreach.cpp

```diff
@@ -339,8 +332,8 @@ void DumpResultsToFile() {
     struct tm* Tm = gmtime(&Time);
     strftime(UtcDateTime, sizeof(UtcDateTime), "%Y.%m.%d-%H:%M:%S", Tm);
     fprintf(File, "%s,%u,%u,%u,%u,%u,%u,%u,%u\n", UtcDateTime,
-        Results.TotalCount, Results.ReachableCount, Results.TooMuchCount, Results.MultiRttCount, Results.RetryCount,
-        Results.IPv6Count, Results.Quicv2Count, Results.WayTooMuchCount);
+        Results.TotalCount.load(), Results.ReachableCount.load(), Results.TooMuchCount.load(), Results.MultiRttCount.load(),
```

Comment:
IMO always specify the intended `memory_order` with atomics

Reply:
Why is the default not acceptable?

Reply:
> **memory_order_seq_cst**	- A load operation with this memory order performs an acquire operation, a store performs a release operation, and read-modify-write performs both an acquire operation and a release operation, plus a single total order exists in which all threads observe all modifications in the same order (see Sequentially-consistent ordering below).

Reply:
You probably want `memory_order_relaxed` which is the opposite end of the spectrum. The same is true for the overloaded `++` operator: using `fetch_and_add` with `memory_order_relaxed` is more efficient and clearer to developers.

Reply:
This is called once for the lifetime of the app. Low perf is fine.